package.json

```diff
@@ -30,11 +30,13 @@
     "@angular/router": "9.1.0",
     "@ng-select/ng-select": "^4.0",
     "@ngneat/until-destroy": "^7.3.2",
+    "@types/luxon": "^1.27.1",
     "bootstrap": "4.3.1",
     "core-js": "2.5",
     "font-awesome": "4.7",
     "jquery": "3.5",
     "lodash": "4.17",
+    "luxon": "^2.0.1",
     "moment": "2.29",
```

Comment:
@jrassa - is the idea to leave `moment` in place for downstream projects until we can all remove it? If not, we should probably remove it from this starter project. Thanks to you and @AedanPettit for working this effort!

Reply:
Yes, that is the idea.  I will prepare a followup PR to take care of that.  There is a little more to it than just removing the dependency.  We added a custom build configuration to exclude moment locale code to help reduce the final bundle size.  With this change, that will no longer be needed.